backend/dev/controller.ts

```diff
@@ -0,0 +1,11 @@
+class Controller {
+  constructor() {
+    this.test();  
+  }
+
+  test() {
+    console.log('+controller');
+  }
```

Comment:
I see no point in a separate method. it is better to make a more explicit message to the console log. something like "the controller is alive!"